travis/publish.sh

```diff
@@ -72,7 +72,7 @@ check_release_tag() {
 }
 
 is_release_commit() {
-  project_version=$(./mvnw help:evaluate -N -Dexpression=project.version|grep -v '\[')
+  project_version=$(./mvnw help:evaluate -N -Dexpression=project.version|sed -n '/^[0-9]/p')
```

Comment:
thx for the heads up.. this pattern only looks at the start of the line, so shouldn't hurt that.